src/containers/IdeLoader.tsx

```diff
@@ -242,8 +243,6 @@ class IdeLoaderContainer extends React.PureComponent<Props, State> {
         currentStep: LoadIdeSteps.INITIALIZING,
         workspaceId: workspace.id,
       });
-    } else if (currentStep === LoadIdeSteps.OPEN_IDE && workspace.status === WorkspaceStatus[WorkspaceStatus.STOPPED]) {
```

Comment:
please clean up the `currentStep` variable declaration: 

https://github.com/eclipse/che-dashboard/pull/193/files#diff-9d36030402d9b9e891ad48f6ffe964d9e83fc5962a3b907a5d0d1928f44776eeR207

Reply:
I have fixed it.